config/initializers/secure_headers.rb

```diff
@@ -72,36 +71,8 @@
   # }
 end
 
-# A tiny middleware that calls a block on each request. When both:
-# 1) the block returns true
-# 2) the response is a 2XX response
-# It deletes the Content-Security-Policy header. This is intended so that we can override
-# SecureHeaders behavior and not set the headers on asset files, because the headers should be set
-# on the document that links to the assets, not the assets themselves.
-class SecureHeaders::RemoveContentSecurityPolicy
```

Comment:
I think we need to keep this class around while we migrate away from the old policy, right?

Reply:
I think this is actually dead code since we moved the acuant SDK into `public`.

Reply:
oh awesome, I forgot we did that yes let's get rid of this! 🔥 🔥 🔥

Reply:
coming all the way back around: We do in fact need this. It looks like SecureHeaders will add a CSP to assets so I put it back.